packages/cli/src/__tests__/commands/Generate.test.ts

```diff
@@ -1,16 +1,24 @@
+import path from 'node:path'
+
 import { BaseContext, jestConsoleContext, jestContext } from '@prisma/get-platform'
 import { ClientEngineType, getClientEngineType } from '@prisma/internals'
-import path from 'path'
 
 import { Generate } from '../../Generate'
+import { promotions } from '../../utils/handlePromotions'
 
 const ctx = jestContext.new().add(jestConsoleContext()).assemble()
 
 describe('using cli', () => {
   it('should work with a custom output dir', async () => {
     ctx.fixture('example-project')
     const data = await ctx.cli('generate')
-    const stdout = data.stdout.replace(/Tip:.*/s, 'Tip: MOCKED RANDOM TIP')
+
+    // Replace any possible entry in `promotions`'s texts with a fixed string to make the snapshot stable
+    const stdout = Object.values(promotions)
+      .map(({ text }) => text)
+      .reduce((acc, curr) => {
+        return acc.replace(new RegExp(`${curr}.*`, 's'), 'Tip: MOCKED RANDOM TIP')
```

Comment:
This may explode with special characters in the tip that are valid regex syntax.

I think this should be enough:

```suggestion
        return acc.replace(curr, 'Tip: MOCKED RANDOM TIP')
```

Reply:
That won't work, as links are appended to it. I can however simplify this by removing the regex altogether. Hold up